src/mon/OSDMonitor.cc

```diff
@@ -4384,6 +4384,22 @@ void OSDMonitor::get_pools_health(
 	detail->push_back(make_pair(HEALTH_WARN, ss.str()));
     }
 
+    if (1 == pool.size) {
```

Comment:
Setting pool size to 1(*not* min_size) may be a deliberate decision. Imposing HEALTH_WARN in that case is only annoyance that users of those pools will ignore.